libs/core/src/lib/feed-input/directives/feed-input-button.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, ElementRef, HostBinding } from '@angular/core';
+
+@Directive({

```

Comment:
What do you think about replacing it to inline component and putting arrow icon inside body. You would be able to support rtl under the hood

Reply:
Do you mean `fd-button`? The main reason why I use a custom directive to disable the button when textarea value is empty. I think we cannot disable through the inline component.

Reply:
@genereg1 now it looks better, with `createArrowIcon`, but could you add RTL support?